config/Dockerfiles/JenkinsfileUnitTests

```diff
@@ -1,9 +1,9 @@
 // Define the openshift pod name, docker repo url, namespace, and service acct
 // for the DSL pod template.
 dslPodName = "contraDsl-${UUID.randomUUID()}"
-dockerRepoURL = env.DOCKER_REPO_URL ?: '172.30.254.79:5000'
-openshiftNamespace = env.OPENSHIFT_NAMESPACE ?: 'continuous-infra'
-openshiftServiceAccount = env.OPENSHIFT_SERVICE_ACCOUNT ?: 'jenkins'
+dockerRepoURL = env.DOCKER_REPO_URL ?: ''
+openshiftNamespace = env.OPENSHIFT_NAMESPACE ?: 'docker.io'
+openshiftServiceAccount = env.OPENSHIFT_SERVICE_ACCOUNT ?: 'fedora:29'
```

Comment:
@14rcole Is there a service account named fedora:29 ???

Reply:
@14rcole Further there is no namespace named docker.io , it should be continous-infra and docker repo url would be docker.io i guess

Reply:
The environment variables are poorly named, but we end up doing a pull of `docker://dockerRepoURL/openshiftNamespace/openshiftServiceAccount` e.g. `docker://172.35.15.12/continuous-infra/jenkins:latest`